README.rst

```diff
@@ -84,16 +84,17 @@ received.
 uplink\_callback
 ''''''''''''''''
 
-The callback function must be declared in your script following this
-structure: \* ``uplink_callback(msg, client)`` \* ``msg``: **JSON ob**
-the message received by the client \* ``client``: **object** the client
-from which the callback is executed are calling
+The callback function must be declared in the script following this
+structure: \* ``uplink_callback(msg, client)`` \* ``msg``: **JSON** the
```

Comment:
JSON is not a type in Python - please mention the type (`dict`/`string`) expected, then mention the JSON nature of that variable